route-rs-runtime/src/lib.rs

```diff
@@ -1,30 +1,21 @@
+//! In brief: The runtime is what takes the basket of computation required by the user, links it together into the desired
+//! graph, and preps it to be handed to the desired runtime for running.
```

Comment:
The desired runtime being Tokio? Or are we saying that the async runtime should be configurable?